### ISSUE TITLE
Git Blame does not match command line's git blame

### ISSUE THREAD

**Issue by lcruz4** [kemayo/sublime-text-git#517]
Doing a git blame on the sublime plugin will give different results than doing a git blame in the command line. It looks like this plugin is trying to ignore moved code, but I'm also seeing weird stuff happen when the same line is found twice.

So if I move a chunk of code, it won't register as a change and still use the blame from before the code was moved. This looks like it was done purposeful, but I for one do not find it useful, and would like to at least have an option to turn it off.

I also have a case where I have the same exact line in two different places in the same file. The blame for both of those lines points to a commit where I added one of the lines. So it doesn't see them as unique lines. I'll can try to reproduce it again.

Both of these discrepancies are different from running git blame on the command line, and the git blame on the command line is correct.

**Comment by yamoinza** [comment]
I agree that this is very confusing and I am unsure of why there would be a difference. I agree that if it is using non-standard options (different from plain "git blame <path-to-file>" on the command line then it should be able to be configured.
